dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -336,3 +338,48 @@ def country_code_to_name(country_code):
         return record[code]['item'][0]['name']
 
     raise requests.exceptions.RequestException(register_response)
+
+
+def unsuspend_suspended_supplier_services(record, suspending_user, client, logger, dry_run):
+    """
+    This method checks if the supplier has services suspended by the suspending_user, if so it un-suspends them. It
+    returns None.
+    :param record: augmented supplier record object generated by countersigning script
+    :param suspending_user: user name of user who performed automated suspension
+    :param client: API client
+    :param logger: logger
+    :param dry_run: bool
+    :return: None
+    :rtype: None
+    """
+    new_service_status, old_service_status = 'published', 'disabled'
+    framework_slug = record['frameworkSlug']
+    supplier_id = record["supplier_id"]
+    suspended_services_on_framework = set(
+        service["id"] for service in
+        client.find_services(
+            supplier_id=supplier_id, framework=framework_slug, status=old_service_status
+        )["services"]
+    )
+    if not suspended_services_on_framework:
+        logger.info(f'Supplier {record["supplier_id"]} has no {old_service_status} services on the framework.')
+
+    services_suspended_by_script = set(
+        event["data"]["serviceId"]
+        for event in
+        client.find_audit_events_iter(
+            audit_type=AuditTypes.update_service_status,
+            data_supplier_id=supplier_id,
+            user=suspending_user
+        )
+    )
+    service_ids = suspended_services_on_framework & services_suspended_by_script
+    # Suspend all services for supplier (the API will de-index the services from search results)
```

Comment:
Is this true?

Reply:
Good question. I am not 100% sure as I took this comment from @lfdebrux's branch https://github.com/alphagov/digitalmarketplace-scripts/blob/83f559f72cc2d04ff2b07de022314e6d04638230/dmscripts/suspend_suppliers_without_agreements.py#L53
A guess @lfdebrux may have more context about where this happens.

Reply:
That comment is present in both `suspend_supplier_services` and `unsuspend_supplier_services`. Is it perhaps a copy-paste error and doesn't make sense here (given that we're not suspending anything)?

Reply:
Good spot. Fixed in 5a7f8ff. I think the edited comment hopefully makes sense as the re-index would need to start indexing these services.